src/main/java/org/broadinstitute/hellbender/engine/filters/CountingReadFilter.java

```diff
@@ -193,7 +221,7 @@ public String getName() {
     /**
      * Private class for Counting OR filters
      */
-    private final class CountingOrReadFilter extends CountingBinopReadFilter {
+    private static final class CountingOrReadFilter extends CountingBinopReadFilter {
```

Comment:
There are no direct tests for the inner ("and"/"or") classes; only indirect ones that verify the results through the public and/or methods. The reason the "and" inner classes are visible for testing is because at some point I added tests that examine the structure of an "and" filter to verify that ordering is preserved during filter execution, because there are usages where read filters are "and-composed" in a specific order for performance reasons. There are no such test for or execution (or any usages of "or" ...AFAIK) since it seemed overkill, though I can add them if you think its worth it.